.gitlab-ci.yml

```diff
@@ -1350,9 +1350,12 @@ workflow:
     when: on_success
   - changes:
       paths:
+        - .gitlab/**/*
+        - omnibus/config/**/*
```

Comment:
Are these leftovers from testing? If they aren't it looks like they should be a lot more precise.

Reply:
This is cherry-picked from https://github.com/DataDog/datadog-agent/pull/24056 so that we include the remote updater e2e tests (which were lacking in the first attempt)

Reply:
Although it shouldn't be part of the PR, I'll rebase after `main` is unfrozen and #24056 is merged